frontend-opal/src/reducers/searchResult.js

```diff
@@ -0,0 +1,10 @@
+const pairs = {
+  SEARCH: (state, action) => action.payload.results
+};
+
+function searchResult(state = {}, action) {
```

Comment:
@hondahmd this is exactly the same in the other, let's just export the pairs and create the reducers in the root